src/events/crawler/_crawl.js

```diff
@@ -1,4 +1,7 @@
+const datetime = require('@architect/shared/datetime/index.js')
 const getSource = require('@architect/shared/sources/_lib/get-source.js')
+const findScraper = require('@architect/shared/sources/_lib/find-scraper.js')
+const findTz = require('../scraper/find-tz/index.js')
```

Comment:
This require felt a bit clumsy, perhaps we should move those files to a new dir, `events/_lib`.  But this will do for now.

Reply:
This won't work. Again, once you're in a function folder, that's its entire known universe. You can't relative path out of this folder.

Reply:
"Again, once you're in a function folder," - this is the first time I've heard that, FYI.